manifests/params_agent.pp

```diff
@@ -1,7 +1,7 @@
 # Wazuh App Copyright (C) 2021 Wazuh Inc. (License GPLv2)
 # Wazuh-Agent configuration parameters
 class wazuh::params_agent {
-  $agent_package_version = '4.2.1-1'
+  $agent_package_version = '4.2.2'
```

Comment:
Add `-1`

Reply:
This change was made in commit https://github.com/wazuh/wazuh-puppet/commit/bfab4c0c1bf761306dc9882808ddcfb44d3d846d